euth/contrib/templates/euth_contrib/includes/pagination.html

```diff
@@ -0,0 +1,33 @@
+{% load i18n contrib_tags %}
+
+{% if is_paginated %}
```

Comment:
is it slightly more efficient, if this condition `is_paginated` is outside of the included template?
Not sure about this, though.

Reply:
Yes, thank you :) Changed it!